images/miq-app/docker-assets/container-scripts/container-deploy-common.sh

```diff
@@ -75,7 +78,7 @@ if [[ -f ${APP_ROOT_PERSISTENT_VMDB}/config/database.yml && -f ${PV_DEPLOY_INFO_
     fi
   fi
 else
-  echo "No pre-existing EVM configuration found on PV"
+  echo "No pre-existing EVM configuration found on region PV"
```

Comment:
Not sure this is enough ....
If region PV data is missing but the Host PV has the data (meaning some kind of error happened and we are in an unfamiliar status).
I think this needs to also be addressed (now that we separated the state into 2 PVs)